etc/kayobe/ansible/requirements.yml

```diff
@@ -1,7 +1,7 @@
 ---
 collections:
   - name: stackhpc.cephadm
-    version: 1.15.1
+    version: 1.17.0
```

Comment:
Any notable changes to add to a reno?

Reply:
I would think these two would be worth a mention which make cephadm key stateless and makes reef default.

https://github.com/stackhpc/ansible-collection-cephadm/pull/145
https://github.com/stackhpc/ansible-collection-cephadm/pull/152

Reply:
We do override the version, so ref change won't affect us.